src/routes/posts.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express')
+
+const db = require('../db')
+
+const router = express.Router()
+
+// CREATE AN ARTICLE
+
+const createPosts = async (req, res) => {
+    try {
+        const { id, userId, title, image, content, published, createdAt } =
+            req.body
+        const newArticle = await db.query(
+            'INSERT INTO posts (id , "userId", title , image , content , published , "createdAt" ) VALUES ($1 , $2 , $3 , $4 , $5 ,$6 , $7) RETURNING *',
+            [id, userId, title, image, content, published, createdAt]
+        )
+        res.json(newArticle)
+    } catch (err) {
+        console.error(err.message)
```

Comment:
@beblicarl  we would want to know the error from the endpoint itself 

e.g frontend can't get access to this console.log to be able to know there's been an error 

make line 9, `async (req, res, next)` and use pass the error to `next` like so

```suggestion
  next(err)
```

PS: You can still keep the logger for your own purposes before the `next(err)` anyway

Reply:
i have worked on this review too, I included the console.error for my own purposes